docs/pages/enroll-resources/kubernetes-access/rancher-integration.mdx

```diff
@@ -0,0 +1,269 @@
+In this step, we'll discover your existing Rancher RBAC configuration to align Teleport with your current access patterns.
+
+ ![Kubernetes access architecture](../../../img/k8s/rancher-cluster-member.png)
+
+To proceed, you'll need to:
+
+1. Authenticate to your cluster with administrative privileges. See guide to [Access a Rancher cluster with kubectl](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig)
+
+2. Discover Rancher RBAC bindings and identifiers:
+- Navigate to **Cluster** in left sidebar
+- Select **More Resources**, Navigate to the **RBAC** tab
+- On the Rbac Tab, select Rolebindings, you'll be presented witha list of Role/Rolebindings with the Kubernetes Rbac subject identifiers.
+
+ ![Kubernetes access architecture](../../../img/k8s/rancher-rbac.png)
+
+Alternatively, you can run the command below to list Kubernetes RoleBindings and extract the group information that Teleport will use to match your existing Rancher access patterns:
+
+   ```code
+   # List all rolebindings with their names and associated groups
+   $ kubectl get rolebindings -n <namespace> -o custom-columns='NAME:.metadata.name,ROLE:.roleRef.name,GROUPS:.subjects[?(@.kind=="Group")].name'
+   
+   # Example output:
+     NAME                     ROLE            GROUPS
+     crtb-w8l2g-cluster-member cluster-member github_team://7415717
+   ```
+Note: This example uses GitHub as the external authentication provider. If your Rancher setup uses a different [external authentication service](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config#external-vs-local-authentication),
+the general approach remains the same. Rancher will return users from the configured external source during role binding and access configuration.
+
+## Step 2/4. Enroll clusters to Teleport
+
+This step connects your Rancher-managed clusters to Teleport for centralized access control:
+
+To enroll your cluster(s), see the official [Teleport Kubernetes Access documentation](/docs/kubernetes-access/getting-started/)
+for detailed enrollment instructions.
+
+## Step 3/4. Configure cluster access in Teleport
+
+Now we'll define Teleport roles that map to Rancher's RBAC permissions:
+
+We'll create a Teleport role that authenticates users to the cluster
+as a member of the `github_team://7415717` group subject assigned the Rancher `cluster-member` Role.
+
+In the previous section, we discovered the built-in Cluster Member role authorizing members
+of this group to perform read-only operations at the cluster level. This specific role:
+- Enables the user to access cluster nodes
+- Denies the user permissions to create, modify or delete resources (e.g. can't deploy workloads)
+
+For more information on permissions granted by the Cluster Member role, see the following guide on
```

Comment:
I would add this paragraph to a "Further reading" section to keep the reader focused on the steps here.